app/scripts/views/components/barChart/barChartTemplate.js

```diff
@@ -41,7 +43,12 @@
           transitionDuration: iViz.opts.dc.transitionDuration
         },
         opts: {},
-        addingChart: false
+        addingChart: false,
+        showDespIcon: true,
```

Comment:
Since showDespIcon is never used locally we could move this chartOperationsHeader.js